dora/core/server/worker/src/test/java/alluxio/worker/dora/DoraMetaManagerTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.dora;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import alluxio.client.file.cache.CacheManager;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.AlluxioProperties;
+import alluxio.conf.InstancedConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.underfs.UfsStatus;
+import alluxio.underfs.UnderFileSystem;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Optional;
+
+public class DoraMetaManagerTest {
+  DoraMetaManager mManager;
+  DoraUfsManager mDoraUfsManager;
+  String mTestMetaStorePath = "./testFolderForMetaStore";
+
+  @Before
+  public void before() throws IOException {
+    AlluxioProperties prop = new AlluxioProperties();
+    prop.set(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR, String.format("%s/metastore",
+        mTestMetaStorePath));
+    AlluxioConfiguration conf = new InstancedConfiguration(prop);
+    PagedDoraWorker worker = mock(PagedDoraWorker.class);
+    CacheManager cacheManager = mock(CacheManager.class);
+    mDoraUfsManager = mock(DoraUfsManager.class);
+    mManager = new DoraMetaManager(conf, worker, cacheManager, mDoraUfsManager);
+  }
+
+  @After
+  public void after() {
+    try {
+      mManager.close();
+    } catch (IOException e) {
+      mManager = null;
+    }
+    File testDir = new File(mTestMetaStorePath);
+    if (testDir.exists()) {
+      deleteDirectory(testDir);
+    } else {
+      throw new RuntimeException("testDir doesn't exist");
```

Comment:
If the directory is not existing, how about do nothing? I'm not sure.

Reply:
if it not existing, somethings must be wrong 🤔

Reply:
So now would you mind delete this method? Actually we don't create any files in the directory.